SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -190,31 +210,17 @@ private Task ProcessPendingErrorsAsync()
         {
             return Task.Run(async () =>
             {
-                var lastSessionErrorLogTimestamp = DateTime.MinValue;
-                ManagedErrorLog lastSessionErrorLog = null;
+                ErrorReport lastSessionErrorReport = null;
                 foreach (var file in ErrorLogHelper.GetErrorLogFiles())
                 {
                     AppCenterLog.Debug(LogTag, $"Process pending error file {file.Name}");
                     var log = ErrorLogHelper.ReadErrorLogFile(file);
 
-                    // Process the file for last session crash report. It doesn't matter if the log is null.
-                    try
-                    {
-                        var otherFileTimestamp = file.LastWriteTime;
-                        if (lastSessionErrorLogTimestamp < otherFileTimestamp)
-                        {
-                            lastSessionErrorLogTimestamp = otherFileTimestamp;
-                            lastSessionErrorLog = log;
-                        }
-                    }
-                    catch (System.Exception ex)
-                    {
-                        AppCenterLog.Warn(LogTag, $"Failed to get the last write time for an error file.", ex);
-                    }
-
                     // Finish processing the file.
                     if (log == null)
                     {
+                        // TODO should we try to see if the name is {guid}.json and call RemoveAllStoredErrorLogFiles when possible? In case json corrupted we should delete exception file as well.
```

Comment:
Please address in this PR

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getfilenamewithoutextension?view=netframework-4.8#System_IO_Path_GetFileNameWithoutExtension_System_String_